internal/pkg/vaultworker/token_test.go

```diff
@@ -0,0 +1,52 @@
+/*
+   Copyright 2019 DELL Technologies.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+  
+  @author: Tingyu Zeng, DELL (created: May 21th, 2019)
+  @version: 1.0.0
+*/
+
+package vaultworker
+
+import (
+	"testing"
+	"fmt"	
+)
+
+const TokenfilepathWin = "..\\..\\..\\test\\test-resp-init.json"
+const TokenfilepathUnix = "../../../test/test-resp-init.json"
```

Comment:
a better way to handle this kind of different implementation/value for different os / arch combinations is to have all the system specific definitions inside a single file with a suffix on the basename for the architecture spec, so for example you would have a simple file like this named `mypack_win.go`:

```go
package mypack

const Tokenfilepath = "blah\\blah\\blah"
```
and for unix have a single file named `mypack_linux.go`:

```go
package mypack

const Tokenfilepath = "blah/blah/blah"
```

Go will automatically "do the right thing" and choose the right file depending on what system is being used

Reply:
also windows supports `/` in filepath names, and furthermore go tests have a `testdata` folder that is ignored by go except for the purposes of testing, I think the `test-resp-init.json` file should live in there, so this could all be simplified by having 
```go
const Tokenfilepath = testdata/test-resp-init.json
```

Reply:
Will cover the unit testing related issue under #78.